config/config.go

```diff
@@ -41,9 +52,18 @@ func NewSDK(opts ...ConfigurationOption) (SDK, error) {
 		o = opt.apply(o)
 	}
 
+	mp, mpShutdown := initMeterProvider(o)
```

Comment:
I think you can considering looking at this code https://github.com/open-telemetry/opentelemetry-go/blob/main/example/dice/otel.go as reference for `NewSDK`.